data/companies.json

```diff
@@ -26,5 +26,19 @@
       "twitter": "https://twitter.com/lftechnology",
       "youtube": "https://www.youtube.com/c/LeapfrogTechnologyInc"
     }
+  },
+  {
+    "name": "Vrit Technologies",
+    "establishedAt": 1546319792,
+    "location": "Sankhamul, Kathmandu",
+    "website": "https://vrittechnologies.com/",
+    "socials": {
+      "facebook": "https://www.facebook.com/vrittech/",
+      "instagram": "https://instagram.com/vrittech",
+      "github": null,
+      "linkedin": "https://www.linkedin.com/company/vrittech/",
+      "twitter": "https://twitter.com/vrittech",
+      "youtube": "https://youtube.com/vrittech"
```

Comment:
The YouTube channel doesn't exist.
![image](https://user-images.githubusercontent.com/60746512/199709756-601d36cf-b594-43bf-9678-4616986d19b9.png)